python/tvm/relay/frontend/onnx.py

```diff
@@ -3477,6 +3493,8 @@ def _get_convert_map(opset):
         "Exp": Renamer("exp"),
         "Greater": Greater.get_converter(opset),
         "Less": Less.get_converter(opset),
+        "LessOrEqual": LessOrEqual.get_converter(opset),
```

Comment:
Can you just use `Renamer`, you are just forwarding the inputs in the converter?

Reply:
E.g. see https://github.com/apache/tvm/pull/8967